darwin/backend_v2.py

```diff
@@ -238,6 +255,12 @@ def import_annotation(
             f"v2/teams/{team_slug}/items/{item_id}/import", payload=payload
         )
 
+    @retry(
+        wait=wait_exponential_jitter(initial=60, max=300),
+        stop=stop_after_attempt(10),
+        retry=retry_if_status_code_429,
+        before_sleep=log_rate_limit_exceeded,
+    )
     @inject_default_team_slug
```

Comment:
any chance we can get a test for this seeing the retries triggered? You probably need to mock the api responses returning `429`s

Reply:
Done